modules/custom-access-flow/strategy.py

```diff
@@ -1,17 +1,56 @@
 from sym.sdk.strategies import AccessStrategy
 from sym.sdk.integrations import slack
+import requests
 
 
 class CustomAccess(AccessStrategy):
+    """This AccessStrategy is fully customizable, and contains basic implementations
+    of all three required methods.
+    """
+
     def fetch_remote_identity(self, user):
+        # If the user doesn't have an identity already for this strategy, use
+        # and save their email address as their identity. This will be returned by
+        # self.get_requester_identity.
```

Comment:
Could also add a comment or example indicating that this is where you would return a custom identity if it exists (e.g. say your internal system has its own user IDs?)